lib/gap_intelligence/models/header.rb

```diff
@@ -0,0 +1,13 @@
+module GapIntelligence
+  class Header < Record
+    attribute :name
+
+    def initialize(attrs = {})
```

Comment:
Honestly I was trying to be consistent with how we did brands...which doesn't have ID either since it's not defined in a database.